<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSegments"/>
   <addaction name="actionArclength"/>
   <addaction name="actionClosest_Point"/>
   <addaction name="actionCurvature"/>
   <addaction name="actionDerivatives"/>
   <addaction name="actionFrenet_Frame"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="settings">
   <property name="minimumSize">
    <size>
     <width>205</width>
     <height>107</height>
    </size>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Spline type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="splineType">
         <item>
          <property name="text">
           <string>B-Spline</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Catmull-Rom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hermite</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bezier</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Arclength step</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="arclengthStep">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="info">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Info</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Num points</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="numPoints"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Length</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="totalLength"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionClosest_Point">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Closest Point</string>
   </property>
  </action>
  <action name="actionDerivatives">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Derivatives</string>
   </property>
  </action>
  <action name="actionFrenet_Frame">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Frenet Frame</string>
   </property>
  </action>
  <action name="actionCurvature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Curvature</string>
   </property>
  </action>
  <action name="actionSegments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Segments</string>
   </property>
  </action>
  <action name="actionArclength">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arclength</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
